docs/manifest.json

```diff
@@ -2296,8 +2296,8 @@
                           "href": "/docs/references/expo/use-local-credentials"
                         },
                         {
-                          "title": "useOAuth()",
-                          "href": "/docs/references/expo/use-oauth"
```

Comment:
❓  Nitpick: double-checking that the removal of this sidebar link was intentional. This looks like a duplicate of a link farther down:

![CleanShot 2024-12-17 at 12 35 44@2x](https://github.com/user-attachments/assets/c6a08447-43bc-48d0-b128-5176180c437b)

Reply:
>do we have a broken link checker?
Yes, we have a lint step, that checks for any links that don't have a corresponding mdx file.